web/coffee-editor-extension/src/browser/coffee-tree/coffee-tree-label-provider-contribution.ts

```diff
@@ -19,44 +19,45 @@ import { CoffeeTreeEditorConstants } from './coffee-tree-editor-widget';
 const DEFAULT_COLOR = 'black';
 
 const ICON_CLASSES: Map<string, string> = new Map([
-    [CoffeeModel.Type.AutomaticTask, 'fa-cog ' + DEFAULT_COLOR],
-    [CoffeeModel.Type.BrewingUnit, 'fa-fire ' + DEFAULT_COLOR],
-    [CoffeeModel.Type.ControlUnit, 'fa-server ' + DEFAULT_COLOR],
-    [CoffeeModel.Type.Decision, 'fa-chevron-up ' + DEFAULT_COLOR],
-    [CoffeeModel.Type.Dimension, 'fa-arrows-alt ' + DEFAULT_COLOR],
-    [CoffeeModel.Type.DipTray, 'fa-inbox ' + DEFAULT_COLOR],
-    [CoffeeModel.Type.Display, 'fa-tv ' + DEFAULT_COLOR],
-    [CoffeeModel.Type.Flow, 'fa-exchange-alt ' + DEFAULT_COLOR],
-    [CoffeeModel.Type.Fork, 'fa-code-branch fa-rotate-90 ' + DEFAULT_COLOR],
-    [CoffeeModel.Type.Join, 'fa-code-branch fa-rotate-270 ' + DEFAULT_COLOR],
-    [CoffeeModel.Type.Machine, 'fa-cogs ' + DEFAULT_COLOR],
-    [CoffeeModel.Type.ManualTask, 'fa-wrench ' + DEFAULT_COLOR],
-    [CoffeeModel.Type.Merge, 'fa-chevron-down ' + DEFAULT_COLOR],
-    [CoffeeModel.Type.Node, 'fa-circle ' + DEFAULT_COLOR],
-    [CoffeeModel.Type.Processor, 'fa-microchip ' + DEFAULT_COLOR],
-    [CoffeeModel.Type.RAM, 'fa-memory ' + DEFAULT_COLOR],
-    [CoffeeModel.Type.Task, 'fa-tasks ' + DEFAULT_COLOR],
-    [CoffeeModel.Type.WaterTank, 'fa-tint ' + DEFAULT_COLOR],
-    [CoffeeModel.Type.WeightedFlow, 'fa-exchange-alt light-orange'],
-    [CoffeeModel.Type.Workflow, 'fa-random ' + DEFAULT_COLOR]
+    [CoffeeModel.Type.AutomaticTask, 'codicon-gear ' + DEFAULT_COLOR],
```

Comment:
Please also update the icon strings here. E.g:
```suggestion
    [CoffeeModel.Type.AutomaticTask, 'gear ' + DEFAULT_COLOR],
```